ci/integTestGen/src/integTestGen.jl

```diff
@@ -0,0 +1,265 @@
+module integTestGen
+
+import Pkg
+import PkgDependency
+import YAML
+
+"""
+    create_working_env(project_path::AbstractString)
+
+Create an temporary folder, setup a new Project.toml and activate it. Checking the dependencies of
+an project only works, if it is a dependency of the integTestGen.jl. Because the package to analyze
+is only a temporary dependency, it should not change permanent the Project.toml of the
+integTestGen.jl. Therefore, the script generates a temporary Julia environment and adds the package
```

Comment:
, it should not change permanent the Project.toml of then itegTestGen.jl. -> , it must not change the `Project.toml` of `integTestGen.jl` permanently.